src/avram/query_builder.cr

```diff
@@ -39,10 +39,13 @@ class Avram::QueryBuilder
 
   private def param_values(params)
     params.values.map do |value|
-      if value.nil?
+      case value
+      when Nil
         nil
-      else
+      when String, UUID
         value.to_s
+      else
```

Comment:
I'm nervous about this because what if in the future we have different types of values? This might be hard to debug in the future. Could we do the opposite like `when JSON::Any -> value.to_json` and the else does `to_s`?

I'm really not sure here since it's been awhile since I looked at this code 🤷‍♂️

Reply:
Yup. I didn't catch this one. Should be same as in the form cast value.